kiali/kiali
# Create controller framework and move validations to offline processing

### Describe the change

Moves calculating namespace/cluster-wide validations into a "controller" for offline processing. Individual object validations are still done "on demand" within the request. Calculates validations every 30s. Does not yet watch istio objects to re-calculate validations when they change.

### Steps to test the PR

1. Visit istio config page and ensure validations are present (configuration is green/yellow/red).
2. Update an individual istio object to be invalid.
3. Return to list page and ensure that validations on the list page match what was reported on the individual page.

### Automation testing

Covered by newly added "controller" tests.

### Issue reference

Relates to #7076
